### PR TITLE
Optional param to call model.eval()

### PR DESCRIPTION
**Pull request by coreyhu**
Adds optional parameter `force_eval=True` to PytorchModelWrapper. Useful since `backward()` with LSTMs must be done in training mode (or with cuda disabled).

### CODE REVIEW

**Review by rshih32 — APPROVED** [review]

File: trulens/nn/models/pytorch.py [hunk 1]
```diff
@@ -337,7 +349,8 @@ def hookfn(self, inpt, outpt):
         with memory_suggestions(device=self.device):
             # Run the network.
             try:
-                self._model.eval()  # needed for determinism sometimes
+                if self.force_eval:
```
Comment (piotrm0) [inline trulens/nn/models/pytorch.py:352]: might no be necessary unless there are some .train() still someplace
